fuzzing/scripts/build/libpng.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Copyright 2018-2020 by
+# Armin Hasitzka and Ben Wagner.
+#
+# This file is part of the FreeType project, and may only be used, modified,
+# and distributed under the terms of the FreeType project license,
+# LICENSE.TXT.  By continuing to use, modify, or distribute this file you
+# indicate that you have read the license and understand and accept it
+# fully.
+
+dir="${PWD}"
+cd $( dirname $( readlink -f "${0}" ) ) # go to `/fuzzing/scripts/build'
+
+path_to_zlib=$( readlink -f "../../../external/zlib" )
+
+path_to_src=$( readlink -f "../../../external/libpng" )
+path_to_build="${path_to_src}/build"
+path_to_install="${path_to_src}/usr"
+
+if [[ "${#}" -lt "1" || "${1}" != "--no-init" ]]; then
+
+    git submodule update --init "${path_to_src}"
+
+    cd "${path_to_src}"
+
+    git clean -dfqx
+    git reset --hard
+    git rev-parse HEAD
+
+    # Based on how oss-fuzz/libpng builds, file at
+    # https://github.com/glennrp/libpng/blob/libpng16/contrib/oss-fuzz/build.sh
```

Comment:
Added a comment here so that if anything ever stops working we know where to look. (It seems my comment in the conversation on the old pull request commit still shows up under the old pull request commit, even if I made that comment while responding to the conversation on the new pull request commit. So mentioning this here so that it is attached to the newer commit.)